tools/klee-ptree/Tree.h

```diff
@@ -0,0 +1,40 @@
+//===-- Tree.h --------------------------------------------------*- C++ -*-===//
+//
+//                     The KLEE Symbolic Virtual Machine
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef KLEE_TREE_H
+#define KLEE_TREE_H
+
+#include "klee/Core/BranchTypes.h"
+
+#include <string>
+#include <vector>
+
+
+/// A Tree node representing a PTreeNode
+struct Node final {
+  std::uint64_t left {0};
+  std::uint64_t right {0};
+  std::uint32_t stateID {0};
+  std::uint32_t asmLine {0};
+  BranchType branchType {BranchType::UNKNOWN};
+  std::uint32_t terminationTypeMask {0};
+};
+
+
+/// An in-memory representation of a complete process tree
```

Comment:
What's wrong with PTree? Also name it something else, maybe StrippedDownPTree?

Reply:
I have to link it and have to allocate nodes I don't need.

Reply:
If you hate linking that much you could also `#include`.  

Surelly allocating a bit more nodes isn't that bad? Having yet another tree implementation that we don't need is bad too. Should also reduce the LOC count making this more mergeable!